src/routes/Home/index.tsx

```diff
@@ -15,7 +19,24 @@ const Card = styled.div`
   margin: 10px;
 `
 
+export const MAX_TXS_DISPLAY = 5
+
 function Home(): ReactElement {
+  const nextTxStore = useSelector(nextTransaction)
+  const queuedTxsStore = useSelector(queuedTransactions)
```

Comment:
`useQueuedTransactions` returns both of these together.

Reply:
You wouldn't need to traverse the data and create a local state if you use this as well.

Reply:
Thanks for bringing that up. I was not aware of that hook